Specifications/Events/for_EventStore/when_committing_a_public_event_with_an_event_type.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Dolittle.SDK.Protobuf;
+using Machine.Specifications;
+using Moq;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Events.for_EventStore
+{
+    public class when_committing_a_public_event_with_an_event_type : given.an_event_store_and_an_execution_context
+    {
+        static an_event content;
+        static EventSourceId event_source;
+        static EventType event_type;
+
+        Establish context = () =>
+        {
+            content = new an_event("goodbye public world", 12345, true);
+            event_source = new EventSourceId("be274a66-e442-455a-b5d9-81d058fc179e");
+            event_type = new EventType("31819551-844f-4a6f-935c-917a126531f6");
+        };
+
+        Because of = async () => await event_store.CommitPublic(content, event_source, event_type);
+        It should_not_call_the_event_types_with_the_content = () => event_types.Verify(_ => _.GetFor(content.GetType()), Times.Never());
+        It should_call_the_converter_with_uncommitted_events = () => converter.Verify(_ => _.ToProtobuf(Moq.It.IsAny<UncommittedEvents>()));
```

Comment:
I would think that we want to verify that the uncommitted event sent in here contains the `content`, `event_source` and `event_type` that is created here at the top?